recipe/build.sh

```diff
@@ -16,17 +16,7 @@ then
     export CXX=g++
 fi
 
-# Doesn't include gmock or gtest. So, need to get these ourselves for `make check`.
-git clone -b release-1.7.0 git://github.com/google/googlemock.git gmock
-git clone -b release-1.7.0 git://github.com/google/googletest.git gmock/gtest
-
-# Build configure/Makefile as they are not present.
-aclocal
-libtoolize
-autoconf
-autoreconf -i
-automake --add-missing
```

Comment:
It's also in the Unix c++ install guide: https://github.com/google/protobuf/blob/master/src/README.md#c-installation---unix